MeshKernelNET/src/MeshKernelNETCore/Api/SplinesToCurvilinearParameters.cs

```diff
@@ -11,15 +11,15 @@ public static SplinesToCurvilinearParameters CreateDefault()
             {
                 AspectRatio = 0.1,
                 AspectRatioGrowFactor = 1.1,
-                AverageWidth = 500.0,
-                CurvatureAdaptedGridSpacing = 1,
-                GrowGridOutside = 1,
+                AverageWidth = 0.005,
```

Comment:
This is a huge difference. Is this intended?

Reply:
Also made coherent with the backend. See ` struct SplinesToCurvilinearParameters` (here)[https://github.com/Deltares/MeshKernel/blob/release/v2.0.0/libs/MeshKernel/include/MeshKernel/Parameters.hpp]. @lucacarniato either the backend or the frontend is wrong. Can you comment?

Reply:
AverageWidth = 500.0 is the correct setting

Reply:
@lucacarniato shared this

![opts](https://github.com/Deltares/MeshKernelNET/assets/121794721/c54ee21b-7ce0-47e3-9f93-3cac40e020b8)

It seems like 500 in the FE is correct. In the BE 0.005 is incorrect and should be adjusted.

Reply:
Done